<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ADB_widget</class>
 <widget class="QWidget" name="ADB_widget">
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QFrame" name="frame">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_7">
      <item>
       <widget class="QGroupBox" name="groupBox_2">
        <property name="minimumSize">
         <size>
          <width>200</width>
          <height>0</height>
         </size>
        </property>
        <property name="title">
         <string>Devices</string>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_5">
         <item>
          <widget class="QComboBox" name="cb_devices"/>
         </item>
         <item>
          <widget class="QPushButton" name="btn_devices">
           <property name="text">
            <string>get</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="groupBox_3">
        <property name="title">
         <string>Screeshot</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_6">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_3">
           <item>
            <widget class="QLabel" name="label_10">
             <property name="text">
              <string>Interval, sec</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QSpinBox" name="sb_interval"/>
           </item>
           <item>
            <widget class="QCheckBox" name="cb_auto">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="text">
              <string>auto</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="btn_create_screenshot">
             <property name="text">
              <string>Shot</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <spacer name="horizontalSpacer_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <layout class="QGridLayout" name="gridLayout">
          <property name="spacing">
           <number>0</number>
          </property>
          <item row="1" column="0">
           <widget class="QToolButton" name="btn_swipe_RL">
            <property name="text">
             <string>...</string>
            </property>
            <property name="icon">
             <iconset resource="../../../../../../lib/pictures/arrow.qrc">
              <normaloff>:/arrow/arrow-left.png</normaloff>:/arrow/arrow-left.png</iconset>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QToolButton" name="btn_swipe_UD">
            <property name="text">
             <string>...</string>
            </property>
            <property name="icon">
             <iconset resource="../../../../../../lib/pictures/arrow.qrc">
              <normaloff>:/arrow/arrow-down.png</normaloff>:/arrow/arrow-down.png</iconset>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QToolButton" name="btn_swipe_DU">
            <property name="text">
             <string>...</string>
            </property>
            <property name="icon">
             <iconset resource="../../../../../../lib/pictures/arrow.qrc">
              <normaloff>:/arrow/arrow-up.png</normaloff>:/arrow/arrow-up.png</iconset>
            </property>
           </widget>
          </item>
          <item row="1" column="2">
           <widget class="QToolButton" name="btn_swipe_LR">
            <property name="text">
             <string>...</string>
            </property>
            <property name="icon">
             <iconset resource="../../../../../../lib/pictures/arrow.qrc">
              <normaloff>:/arrow/arrow-right.png</normaloff>:/arrow/arrow-right.png</iconset>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QGroupBox" name="groupBox">
        <property name="title">
         <string>Tap</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <item>
          <layout class="QGridLayout" name="gridLayout_2">
           <item row="0" column="0">
            <widget class="QLabel" name="label">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="text">
              <string>X:</string>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QLabel" name="label_2">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="text">
              <string>Y:</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QSpinBox" name="sb_pos_x"/>
           </item>
           <item row="1" column="1">
            <widget class="QSpinBox" name="sb_pos_y"/>
           </item>
           <item row="0" column="2" rowspan="2">
            <widget class="QPushButton" name="btn_tap">
             <property name="text">
              <string>Tap</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QGroupBox" name="groupBox_4">
           <property name="title">
            <string>Commands</string>
           </property>
           <layout class="QHBoxLayout" name="horizontalLayout_6">
            <item>
             <widget class="QComboBox" name="cb_commands"/>
            </item>
            <item>
             <widget class="QToolButton" name="btn_adb">
              <property name="text">
               <string>...</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <widget class="QLineEdit" name="le_cmd"/>
        </item>
        <item>
         <widget class="QToolButton" name="btn_cmd">
          <property name="text">
           <string>...</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QPushButton" name="btn_test">
        <property name="text">
         <string>test</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QFrame" name="frame_test">
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout">
           <item>
            <widget class="QToolButton" name="btn_1">
             <property name="text">
              <string>1</string>
             </property>
             <property name="icon">
              <iconset resource="../../../../../../lib2/icons/digits/digits.qrc">
               <normaloff>:/red/1.png</normaloff>:/red/1.png</iconset>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QToolButton" name="btn_2">
             <property name="text">
              <string>2</string>
             </property>
             <property name="icon">
              <iconset resource="../../../../../../lib2/icons/digits/digits.qrc">
               <normaloff>:/red/2.png</normaloff>:/red/2.png</iconset>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QToolButton" name="btn_3">
             <property name="text">
              <string>3</string>
             </property>
             <property name="icon">
              <iconset resource="../../../../../../lib2/icons/digits/digits.qrc">
               <normaloff>:/red/3.png</normaloff>:/red/3.png</iconset>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QToolButton" name="btn_4">
             <property name="text">
              <string>4</string>
             </property>
             <property name="icon">
              <iconset resource="../../../../../../lib2/icons/digits/digits.qrc">
               <normaloff>:/red/4.png</normaloff>:/red/4.png</iconset>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QSpinBox" name="sb_port"/>
           </item>
           <item>
            <widget class="QToolButton" name="btn_start">
             <property name="text">
              <string>...</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QToolButton" name="btn_stop">
             <property name="text">
              <string>...</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../../../../../../lib/pictures/arrow.qrc"/>
  <include location="../../../../../../lib2/icons/digits/digits.qrc"/>
 </resources>
 <connections/>
</ui>
